core/src/main/resources/lib/layout/layout.jelly

```diff
@@ -189,7 +189,7 @@ THE SOFTWARE.
     <!-- for accessibility, skip the entire navigation bar and etc and go straight to the head of the content -->
     <a href="#skip2content" class="skiplink">Skip to content</a>
 
-    <div id="page-head">
+    <div id="page-head" role="heading">
```

Comment:
`<l:pageHeader>` already contains a `<header>` tag if that's what you are looking for. That said, AFAIK the `heading` role is applied to headlines and titles, not blocks. So this change would be invalid either way.
```suggestion
    <div id="page-head">
```